.circleci/config.yml

```diff
@@ -15,3 +15,6 @@ jobs:
       - run: go vet ./...
       - run: CGO_ENABLED=0 go test ./cmd/... -count=1 -p=1
       - run: CGO_ENABLED=0 go test ./pkg/... -count=1 -p=1
+      - run: git clone https://github.com/bats-core/bats-core.git
```

Comment:
@spolti It seems that the circle ci job is only running the `build` job (https://circleci.com/gh/kiegroup/kogito-cloud-operator/783#build-parameters/containers/0)
So for now, I included it into the `build` job, but I would like better to have a second job, like `bats-tests` job.
Do you know how we can update that ?

Reply:
@radtriste for use multiple jobs, then a workflow is necessary:

https://circleci.com/docs/2.0/workflows/

but, in this case, I think that a sequential steps is ok, or, is there a specific reason to use multiple jobs for it?

Reply:
If you are ok with the current solution, we can stick to it (and refactor later if needed)

Reply:
It is ok for me